service.go

```diff
@@ -52,11 +68,20 @@ func (s *topicServiceImpl) getTopicByUUID(uuid string) (topic, bool) {
 	return topic, found
 }
 
-func (s *topicServiceImpl) initTopicsMap(terms []term) {
-	for _, t := range terms {
-		top := transformTopic(t)
+func (s *topicServiceImpl) checkConnectivity() error {
+	// TODO: Can we just hit an endpoint to check if TME is available? Or do we need to make sure we get genre taxonmies back? Maybe a healthcheck or gtg endpoint?
+	_, err := s.repository.GetTmeTermsFromIndex(1)
```

Comment:
I wonder whether we should just check the host is up - I have this feeling that the TME service isn't going to cope with a lot of load, and our healthcheck will put it under load - each deployed transformer making a request once a minute. Going to be a request a second. Or - we move to an asynchronous healthcheck where we run the check less often, but report on it when the healthcheck is called each time. Let's discuss.